tooling/cli/Cargo.toml

```diff
@@ -92,6 +92,7 @@ libc = "0.2"
 lto = true
 
 [features]
-default = [ "tauri-bundler/default" ]
+default = [ "tauri-bundler/rustls" ]
```

Comment:
i guess both work but i still feel like this should be this right?
```suggestion
default = [ "rustls" ]
```

Reply:
Yeah, thanks (damn I didn't see this as a suggestion and manually commited it)